hdk/cl/examples/cl_uram_example/build/constraints/cl_pnr_user.xdc

```diff
@@ -1,10 +1,10 @@
 # This contains the CL specific constraints for Top level PNR
 
 # False path between vled on CL clock and Shell asynchronous clock
-set_false_path -from [get_cells WRAPPER_INST/CL/vled_q_reg*] 
 
-# False paths between main clock and tck
```

Comment:
Please keep the existing false path constraints and add new generated clock constraints.